### ISSUE TITLE
[Bug] Weird YouTube logo on app opening

### ISSUE THREAD

**Issue by Princevs2** [qnblackcat/uYouPlus#194]
### Device info
- iOS/iPadOS version: ios 15.5
- Device model: iphone 13
- Sideload tool (Ex: AltStore, Sideloadly...) Altstore
- Version of uYouPlus: 17.22.1 v2.1

### Describe the bug

I have installed the latest uYouplus 17.22.1 v2.1. 
when i launch the app the logo looks wierd & uncooked.  
also it would be good if you give an option to hide the sponser button. 
screenshot added. 

### How to reproduce the bug
_(Please attach videos or screenshots if  @possible)_
![73ACD5AF-FF96-489F-9813-AF590772197A](https://user-images.githubusercontent.com/85836097/172394940-373e09ff-fab0-4bf9-99a4-42f271e07dfc.png)
![Uploading 8A6F08BB-D8B4-4441-9605-B4E021691668.jpeg…]()

### Crash log (if possible)
_(The crash log can be found in the  Settings app - Privacy - Analytics & Improvements - Analytics Data - Youtube...)_

**Comment by Princevs2** [comment]
![Uploading DA54E4E4-5076-42E8-8F80-7FEF3ACBB2AB.jpeg…]()

**Comment by qnblackcat** [comment]
> ...also it would be good if you give an option to hide the sponser button.

then how will you access the setting of iSponsorBlock?!

**Comment by Princevs2** [comment]
I don't use Isponsership sir. 
I have disabled it already, just want an option to hide that button!
Also can you plz fix that youtube logo in next update. 
Make it look like original Youtube. 
Sometimes Ui feels stuttering, I'm just informing you so that you can work on it. 
Let me tell you one more thing,
I have used many modded Youtube clients but this one is the best!
Thank you so much. ❤️> > ...also it would be good if you give an option to hide the sponser button.
> 
> 
> 
> then how can you access the setting of iSponsorBlock?!

**Comment by arichorn** [comment]
> I have installed the latest uYouplus 17.22.1 v2.1.
when i launch the app the logo looks wierd & uncooked.

im pretty sure the dev knows about the bug, it’ll be fine

**Comment by qnblackcat** [comment]
> … if you give an option to hide the iSponsorBlock button.

I will not add an option like this to uYou+. In case you hate iSponsorBlock that much, there are several _workarounds_:
1. Extract the iPA, go to `Payload > YouTube.app > Frameworks`, delete the `iSponsorBlock.dylib`, `libcolorpicker.dylib`, `Alderis.framework`, then re-pack the iPA.

2. Add this code and compile your own IPA of uYou+:
```
@interface YTQTMButton : UIView
@end
@interface YTRightNavigationButtons : UIView
@property (nonatomic, strong, readwrite) YTQTMButton *sponsorBlockButton;
@end

%hook YTRightNavigationButtons
- (void)didMoveToWindow {
    %orig;
    self.sponsorBlockButton.hidden = YES;
}
%end
```

> ...can you plz fix that youtube logo in next update.

I can not reproduce the issue at all. Have you tried to reinstall uYou+?

**Comment by arichorn** [comment]
> I can not reproduce the issue at all. Have you tried to reinstall uYou+?

This issue happened in 17.21.3-2.1
It must’ve been caused by the IAmYouTube Tweak

**Comment by JackPlaysGames0510** [comment]
If you restart your device delete the app then install it again it should be fine

**Comment by qnblackcat** [comment]
@Princevs2 have you tried the solution above?

**Comment by qnblackcat** [comment]
Closed due to inactive

**Comment by arichorn** [comment]
The reason why it’s not active is because this issue was fixed way back in uYouPlus 17.23.6-2.1-F2
